agent/engine/docker_task_engine_linux_test.go

```diff
@@ -198,6 +206,8 @@ func TestDeleteTask(t *testing.T) {
 	assert.Len(t, attachments, 1)
 
 	taskEngine.deleteTask(task)
+	_, resourceValueNotRemoved := asmSecretResource.GetCachedSecretValue("CachedValue")
```

Comment:
nit: This variable name is confusing. I think it should be something like `resourceValueExists`.
Otherwise the below line (`assert.False(t, resourceValueNotRemoved)`) reads like "assert that the resource value was not not removed"